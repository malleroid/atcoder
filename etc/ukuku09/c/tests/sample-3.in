10 13 10
#############
#vvv>vvv#v<.#
#.^.^<#>.v<.#
#.#>v<<<>>v.#
#^v<.v^v>#<<#
#^.>vv^.^.^^#
#<><vv>^vv^>#
##<.>^>v.^<<#
#.v.^v.>><v^#
#############
10 4 8 6
11 7 11 5
8 9 7 8
12 4 5 6
5 6 8 6
12 3 11 7
2 5 8 9
10 4 6 8
4 3 4 3
7 7 3 6
