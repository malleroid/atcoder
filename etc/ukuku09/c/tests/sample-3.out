-1
2
3
-1
10
3
-1
-1
0
9
